docs/timeseries_pipeline_elements.md

```diff
@@ -1,4 +1,4 @@
-# Timevector Pipelines [<sup><mark>experimental</mark></sup>](/docs/README.md#tag-notes)
```

Comment:
This is a bit different than the pipelined accessors that we stabilized in 1.10.1.  I think this should still be marked experimental.  Hmm, taking a look this doc also looks terribly out of date as it only has three pipeline elements (I think we're closer to dozen at this point).